timestampvm/vm.go

```diff
@@ -346,7 +346,7 @@ func (vm *VM) Version() (string, error) {
 	return Version.String(), nil
 }
 
-func (vm *VM) Connected(id ids.ShortID) error {
+func (vm *VM) Connected(id ids.ShortID, nodeVersion version.Application) error {
```

Comment:
I updated the Connected method to pass nodeVersion around. Sorry I missed this dependency